app/pb_kits/playbook/pb_user/_user.jsx

```diff
@@ -2,50 +2,81 @@
 
 import React from 'react'
 import classnames from 'classnames'
+import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
 import { Avatar, Body, Title } from '../'
 import { globalProps } from '../utilities/globalProps.js'
 
 type UserProps = {
-  className?: string,
-  id?: string,
-  name: string,
-  territory?: string,
-  title?: string,
-  size?: "sm" | "md" | "lg",
   align?: "left" | "center" | "right",
+  aria?: object,
+  avatar?: Boolean,
+  avatarUrl?: String,
+  className?: String,
+  dark?: boolean,
+  data?: object,
+  id?: String,
+  name?: String,
   orientation?: "horiztonal" | "vertical",
-  avatar?: boolean,
-  avatarUrl?: string,
+  size?: "sm" | "md" | "lg",
+  territory?: String,
+  title?: String,
 }
 
 const User = (props: UserProps) => {
   const {
-    name = '',
-    territory = '',
-    title = '',
     align = 'left',
-    orientation = 'horizontal',
-    size = 'sm',
+    aria = {},
     avatar = false,
     avatarUrl,
+    className,
+    dark = false,
+    data = {},
+    id,
+    name,
+    orientation = 'horizontal',
+    size = 'sm',
+    territory = '',
+    title = '',
   } = props
+
+  const avatarSizeMap = {
+    lg: 'xl',
+    md: 'md',
+    sm: 'sm',
+  }
+
+  const ariaProps = buildAriaProps(aria)
+  const dataProps = buildDataProps(data)
+
+  const classes = classnames(
+    buildCss('pb_user_kit', align, orientation, size),
+    globalProps(props),
+    className,
+  )
+
   return (
-    <div className={classnames(`pb_user_kit_${align}_${orientation}_${size}`, globalProps(props))}>
+    <div
+        {...ariaProps}
+        {...dataProps}
+        className={classes}
+        id={id}
+    >
       <If condition={avatar || avatarUrl}>
         <Avatar
             imageUrl={avatarUrl}
             name={name}
-            size={size}
+            size={avatarSizeMap[size]}
         />
       </If>
 
       <div className="content_wrapper">
         <Title
+            dark={dark}
             size={size == 'lg' ? 3 : 4}
             text={name}
         />
         <Body color="light">
-          {territory == '' ? title : `${territory} • ${title}`}
+          {territory === '' ? `${title}` : `${territory} • ${title}`}
```

Comment:
This really wasn't needed. `title` by itself would have worked just fine fwiw.

Reply:
Thanks!